bears/general/FilenameBear.py

```diff
@@ -98,6 +101,13 @@ def run(self, filename, file,
                 'Filename does not use the suffix {!r}.'.format(
                     filename_suffix))
 
+        p = platform.system()
+        max_len = max_len_path_windows if p == 'Windows' else max_len_path_unix
+        if len(new_name) > max_len-len(extension):
+            messages.append(
+                'Filename length is more than {}, so it is too long.'.format(
```

Comment:
1. you should be precise with path vs filename
2. incorporate the actual length, I would recommend
    ```
    Filename length ({}) is more than {}.
    ```
    or even better imo
    ```
    'Filename is too long ({} > {})'
    ```
    but up to you to phrase a nice message ;)
3. "so it is too long" is redundant ;)

Reply:
Oh maybe you should also incorporate the system the user is on, so he isn't confused when he sees "Wait, my filename is too long? But `filename_length_unix` is 1000", and because he's on Windows he oversees that the windows-default is used ;) (unlikely, but just a suggestion to provide more information in the message)

Reply:
I think, that this message is great :) 
> 'Filename is too long ({} > {})'